### PR TITLE
Gpt4 tts

### PR DESCRIPTION
**Pull request by rjguedes8**

### CODE REVIEW

File: gpt4-tts/walk_description.ipynb
[the remark this answers is not in the document]
Comment (rjguedes8) [reply gpt4-tts/walk_description.ipynb:1]: I'm not uploading the video, do you want me to?

[the remark this answers is not in the document]
Comment (rjguedes8) [reply gpt4-tts/walk_description.ipynb:1]: in the .env file, do you want to add just the OPEN AI KEY or also the paths to files

File: gpt4-tts/requirements.txt
[the remark this answers is not in the document]
Comment (rjguedes8) [reply gpt4-tts/requirements.txt:1]: no specifics about the video, anything should work
done

File: gpt4-tts/walk_description.ipynb
[the remark this answers is not in the document]
Comment (rjguedes8) [reply gpt4-tts/walk_description.ipynb:1]: the .env file is not there due to git ignore, let mw know if you want me to upload it too

[the remark this answers is not in the document]
Comment (luisroque) [reply gpt4-tts/walk_description.ipynb:1]: No, you did it correctly. It was just to change getting the video from data/ as you did and all the env variables from .env. The data and .env should be in the gitignore, all good!

**Review by luisroque — APPROVED** [review]

**Pull request merged**